dev/NavigationView/TestUI/NavigationViewPage.xaml

```diff
@@ -52,6 +52,12 @@
                     </muxcontrols:NavigationViewItem.Content>
                 </muxcontrols:NavigationViewItem>
                 <muxcontrols:NavigationViewItem x:Name="IntegerItem" AutomationProperties.Name="Integer" Icon="Accept"/>
+                <muxcontrols:NavigationViewItemSeparator />
+                <muxcontrols:NavigationViewItem x:Name="HasChildItem" AutomationProperties.Name="HasChildItem" Content="Item with child">
+                    <muxcontrols:NavigationViewItem.MenuItems>
+                        <muxcontrols:NavigationViewItem x:Name="ChildItem" AutomationProperties.Name="ChildItem" Content="Child Item Content" Icon="Document" />
+                    </muxcontrols:NavigationViewItem.MenuItems>
+                </muxcontrols:NavigationViewItem>
```

Comment:
alright I debugged though this and figured out why the test is failing. On RS2 ScrollViewer handles the down arrow key and scrolls the pane instead of moving focus, this wasn't an issue before this change because all of the nav view items fit on the screen without scrolling, but with these added they no longer fit and now the arrow key doesn't move focus and instead scrolls.  This scrollviewer behavior is known on RS2 and changed for RS3, I would just disable the test on RS2.

Reply:
@Felix-Dev sorry it took so long for me to take a look.

Reply:
Thanks for taking a look here, @StephenLPeters! Will add a commit disabling this test on RS2 as you've suggested soon.

Reply:
And I just saw you already create such a commit 😀

Reply:
yes please!

Reply:
Merged with master now.

Reply:
@StephenLPeters Both other failing tests also use the down arrow key so they need to be disabled on RS2 and below as well. Are we fine with disabling two more tests on RS2 (which is still supported by WinUI) and if yes, do you want to add that code or should I do it?

Reply:
These ones seem like we could focus the new item using something other than the arrow key, maybe just for rs2.

Reply:
@StephenLPeters I'm not yet sure what to do with the `CommonTests.MenuItemKeyboardInvokeTest()`: [That test](https://github.com/microsoft/microsoft-ui-xaml/blob/master/dev/NavigationView/NavigationView_InteractionTests/CommonTests.cs#L693) on RS2 and below already "only" tests that pressing the down arrow key moves focus from the first item to the second item in the NavigationView. Setting the focus on the second item via different means on RS2 seems to defeat the purpose of this test on RS2. However, disabling this test on RS2 because of an unrelated test page modification is not great either. Thus, I see two options here: Either moving that test to use a different test page or moving my added interaction test to a different page.

There already is a test page explicity named "NavigationVew Regression Test" with a layout like this:
![image](https://user-images.githubusercontent.com/1398851/88338249-6257b800-cd38-11ea-9e46-134cb7f5ec3e.png)
which should work for the `MenuItemKeyboardInvokeTest()` test. That test is also set up with code like 
```cs
var testScenarios = RegressionTestScenario.BuildLeftNavRegressionTestScenarios();
```
so it appears to me it would be a good fit for that test page anyway.

As for `SelectionTests.MenuItemAutomationSelectionTest()`: I could change the code
```cs
Log.Comment("Move focus to the second item by pressing down(left nav)/right(right nav) arrow once");
var key = Key.Right;
if (testScenario.IsLeftNavTest)
{
    key = Key.Down;
}
KeyboardHelper.PressKey(key);
Wait.ForIdle();
Verify.IsTrue(secondItem.HasKeyboardFocus);
```
to 
```cs
// On RS2 and below scrollviewer handles arrow keys and thus a scroll instead of a move focus happens. To work around this,
// we explicitly set focus on RS2 and below
if (!PlatformConfiguration.IsOsVersionGreaterThanOrEqual(OSVersion.Redstone3))
{
    secondItem.SetFocus();
    Wait.ForIdle();
    Verify.IsTrue(secondItem.HasKeyboardFocus);
}
else
{
    Log.Comment("Move focus to the second item by pressing down(left nav)/right(right nav) arrow once");
    var key = Key.Right;
    if (testScenario.IsLeftNavTest)
    {
        key = Key.Down;
    }
    KeyboardHelper.PressKey(key);
    Wait.ForIdle();
    Verify.IsTrue(secondItem.HasKeyboardFocus);
}            
```
which will still keep that test relevant on RS2. However, I think we could alos move that test to use a different test page such as suggested for the `CommonTests.MenuItemKeyboardInvokeTest()` above.

What do you you think?

Reply:
I suppose seperating theses tests onto different pages is the correct solution. We should also make a note on the page that ends up running these tests on RS2 that we cannot add more items to the NavView without breaking the test, because of this scrollviewer issue.